From fb735d5ddc611d1cedeb81a67b679e94268a7a62 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Thu, 5 May 2022 18:35:03 +0200
Subject: [PATCH] Revert "ASoC: Intel: sof_es8336: Add a quirk for Huawei
 Matebook D15"

This reverts commit de7658cceecd1bc4fb1c8aa979e9d0ef689af609.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 sound/soc/intel/boards/sof_es8336.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/sound/soc/intel/boards/sof_es8336.c b/sound/soc/intel/boards/sof_es8336.c
index b18951a8f..28d7670b8 100644
--- a/sound/soc/intel/boards/sof_es8336.c
+++ b/sound/soc/intel/boards/sof_es8336.c
@@ -252,15 +252,6 @@ static const struct dmi_system_id sof_es8336_quirk_table[] = {
 					SOF_ES8336_TGL_GPIO_QUIRK |
 					SOF_ES8336_ENABLE_DMIC)
 	},
-	{
-		.callback = sof_es8336_quirk_cb,
-		.matches = {
-			DMI_MATCH(DMI_SYS_VENDOR, "HUAWEI"),
-			DMI_MATCH(DMI_BOARD_NAME, "BOHB-WAX9-PCB-B2"),
-		},
-		.driver_data = (void *)(SOF_ES8336_HEADPHONE_GPIO |
-					SOC_ES8336_HEADSET_MIC1)
-	},
 	{}
 };
 
-- 
2.36.0.44.g0f828332d5

